### PR TITLE
Use Docker Login For Public ECR Instead Of AWS ECR Login

### PR DESCRIPTION
**Pull request by sethAmazon**
**Description:** 
Can't log in to public ecr

### CODE REVIEW

**Review by aateeqi — APPROVED** [review]

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -523,6 +523,10 @@ jobs:
         if: steps.e2etest-release.outputs.cache-hit != 'true'
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+        with:
```
Comment (jefchien) [inline .github/workflows/CI.yml:526]: I don't think this action takes the credentials. https://github.com/aws-actions/amazon-ecr-login#credentials-and-region
That's done by `aws-actions/configure-aws-credentials@v1`, which happens in an earlier step.

**Comment by jefchien** [comment]
Perhaps we can use https://github.com/docker/login-action#aws-public-elastic-container-registry-ecr
```
uses: docker/login-action@v1
with:
  registry: public.ecr.aws
  username: ${{ secrets.AWS_ACCESS_KEY_ID }}
  password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
env:
  AWS_REGION: <region>
```

File: .github/workflows/CI.yml (hunk 1 above)
Comment (khanhntd) [reply .github/workflows/CI.yml:526]: True. I think you are using this instead of aws-actions/amazon-ecr-login@1 aws-actions/amazon-ecr-login@v1

**Review by khanhntd — APPROVED** [review]
LGTM.

**Review by jefchien — APPROVED** [review]

**Pull request merged**